packages/markup-frontstage/.gitignore

```diff
@@ -0,0 +1,42 @@
+# Logs
+*.log

```

Comment:
No sir, added that just for DS store which didn't worked from root because I had spelling mistake, Carl found it and changed it on both but forgot to remove from makrup-frontstage, for now it's removed please :)